### PR TITLE
Add test language detection

### PR DESCRIPTION
**Pull request by Ydrasil**

### CODE REVIEW

File: blog/test.py [hunk 1]
```diff
@@ -20,3 +20,14 @@ def test_content_cleaned_html(self):
         article = Article(title='',
                           content='<p><em>emphased content</em><i>also emphased</i></p>').save()
         self.assertEqual(article.content, '<p><em>emphased content</em><em>also emphased</em></p>')
+
+    def test_language_detection(self):
+        article = Article(title='Un titre en français',
+                          content='<p>Voici le contenu de l\'article qui est '
+                                  'lui aussi en français</p>').save()
+        self.assertTrue(article.language == 'fr')
```
Comment (vinyll) [inline blog/test.py]: use `self.assertEqual`

File: blog/test.py [hunk 2]
```diff
@@ -20,3 +20,14 @@ def test_content_cleaned_html(self):
...
+
+        article = Article(title='A title in english',
+                          content='<p>The content of the article is in '
+                                  'english</p>').save()
+        self.assertTrue(article.language == 'en')
```
Comment (vinyll) [inline blog/test.py]: Same here

File: blog/test.py [hunk 3]
```diff
@@ -20,3 +20,14 @@ def test_content_cleaned_html(self):
...
                           content='<p><em>emphased content</em><i>also emphased</i></p>').save()
         self.assertEqual(article.content, '<p><em>emphased content</em><em>also emphased</em></p>')
+
+    def test_language_detection(self):
+        article = Article(title='Un titre en français',
+                          content='<p>Voici le contenu de l\'article qui est '
+                                  'lui aussi en français</p>').save()
+        self.assertTrue(article.language == 'fr')
+
+        article = Article(title='A title in english',
```
Comment (vinyll) [inline blog/test.py]: Capital _e_ in English ;)

File: blog/test.py [hunk 4]
```diff
@@ -20,3 +20,14 @@ def test_content_cleaned_html(self):
...
+                          content='<p>The content of the article is in '
+                                  'english</p>').save()
```
Comment (vinyll) [inline blog/test.py]: Same _E_

**Review by vinyll — CHANGES_REQUESTED** [review]

**Comment by Ydrasil** [comment]
@vinyll Done :+1:

**Review by vinyll — APPROVED** [review]

**Pull request merged**